### PR TITLE
Add Account Feature Flag

### PR DESCRIPTION
**Pull request by lvndry**
### 📝 Description

Add Feature flag for all add Account entrypoints

### ❓ Context

- **Impacted projects**: `LLD/LLM`
- **Linked resource(s)**:
  - https://ledgerhq.atlassian.net/browse/LIVE-5099
  - https://ledgerhq.atlassian.net/browse/LIVE-5100
  - https://ledgerhq.atlassian.net/browse/LIVE-5110

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx [hunk 1]
```diff
@@ -14,6 +15,10 @@ export const useFirebaseRemoteConfig = () => useContext(FirebaseRemoteConfigCont
 
 export const formatToFirebaseFeatureId = (id: string) => `feature_${snakeCase(id)}`;
 
+export const formatCurrencyIdToFeatureKey = (id: string) => {
+  return `currency${startCase(id).replace(/\s/g, "")}`;
```
Comment (henrily-ledger) [inline apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx:19]: Just to understand, nothing mandatory, why do we use startCase and not SnakeCase ?

File: apps/ledger-live-mobile/src/components/FirebaseRemoteConfig.tsx [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 import { defaultFeatures } from "@ledgerhq/live-common/featureFlags/index";
-import { reduce, snakeCase } from "lodash";
+import { reduce, snakeCase, startCase } from "lodash";
 import { DefaultFeatures } from "@ledgerhq/types-live";
 
 export const formatToFirebaseFeatureId = (id: string) =>
   `feature_${snakeCase(id)}`;
 
+export const formatCurrencyIdToFeatureKey = (id: string) => {
+  return `currency${startCase(id)}`;
```
Comment (henrily-ledger) [inline apps/ledger-live-mobile/src/components/FirebaseRemoteConfig.tsx:11]: We do not have the same definition on LLD and LLM ? is that expected ?

**Review by henrily-ledger — CHANGES_REQUESTED** [review]
I'm not sure to see code change on mobile part ?

File: apps/ledger-live-desktop/src/renderer/drawers/DataSelector/AccountList.tsx [hunk 3]
```diff
@@ -61,11 +69,12 @@ export function AccountList({ currency, onAccountSelect, accounts$ }: Props) {
                 marginLeft: 12,
               }}
             >
-              {"Add account"}
+              {t("addAccounts.title")}
```
[the remark this answers is not in the document]
Comment (lvndry) [reply apps/ledger-live-desktop/src/renderer/drawers/DataSelector/AccountList.tsx:72]: Yes

File: apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx (hunk 1 above)
Comment (lvndry) [reply apps/ledger-live-desktop/src/renderer/components/FirebaseRemoteConfig.tsx:19]: Yeah would be better to use snake_case everywhere but remotely the feature flags are snake_case and locally they are in camelCase

So in remote it's `feature_currency_avalanche_c_chain` and in the code it's `currencyAvalancheCChain`

**Review by henrily-ledger — APPROVED** [review]

**Pull request merged**